src/containers/Bookings.js

```diff
@@ -1,18 +1,62 @@
 import React, { Component } from "react";
 import Search from "../components/Search.js";
+import ResultsTable from '../components/ResultsTable'
 // import Results from "../components/Results.js";
-// import * as FakeBookings from "../data/fakeBookings.json";
+import FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false,
+      bookings: FakeBookings,
+      selected  : false
+    };
+  }
   search = () => {
     console.info("to do!");
   };
-  
+
+  handleBookings(){
+    if(this.state.isClicked===true){
+      return this.state.bookings
+    }else {
+      return [];
+    }
+  }
+
+  onClicked = () => {
+    this.setState({
+     isClicked: true,
+   });
+  }
+  sortIt = (thing) => {
+    console.log('thing:', thing)
+    const bookings = this.state.bookings
+    const sortedBookings = this.state.bookings.sort()
```

Comment:
You need to filter by the particular header that is clicked, there is no single way to sort a data table